app/controllers/posts_controller.rb

```diff
@@ -11,6 +11,9 @@ def show
   def create
     params.inspect
     @post = Post.create(post_params)
```

Comment:
is this necessary given line 14?

Reply:
what about line 13? are they both the same?

Reply:
I've left line 14 and  got rid of 13 instead.